tests/filesystem.go

```diff
@@ -109,6 +110,27 @@ func pickPartition(t *testing.T, device string, partitions []*types.Partition, l
 	return ""
 }
 
+func writeIgnitionProfile(t *testing.T, path string, partitions types.Partitions, oemDirs []string) {
+	var profile struct {
```

Comment:
Can we move the struct definition into `tests/types`?

Reply:
Nothing outside this function should ever have to use it, but I can move it if you'd like.

Reply:
My thought was that if we exposed this in the `types` module the tests could later expand the logic to allow for an individual test to specify a custom profile (potentially one where the `OEM.Device` is something other than the `OEM` partition on drive 0. I'm okay punting and letting that be part of a later refactor if we ever want to add a case like that.